great_expectations/core/usage_statistics/anonymizers/profiler_run_anonymizer.py

```diff
@@ -0,0 +1,262 @@
+
+        return anonymized_domain_builder
+
+    def _anonymize_parameter_builders(
+        self, parameter_builders: List[dict]
+    ) -> List[dict]:
+        anonymized_parameter_builders: List[dict] = []
+
+        for parameter_builder in parameter_builders:
+            anonymized_parameter_builder: dict = self._anonymize_parameter_builder(
+                parameter_builder
+            )
+            anonymized_parameter_builders.append(anonymized_parameter_builder)
+
+        return anonymized_parameter_builders
+
+    def _anonymize_parameter_builder(self, parameter_builder: dict) -> dict:
+        anonymized_parameter_builder: dict = self.anonymize_object_info(
+            object_config=parameter_builder,
+            anonymized_info_dict={},
+            ge_classes=self._ge_parameter_builders,
+            runtime_environment={
+                "module_name": "great_expectations.rule_based_profiler.parameter_builder"
+            },
+        )
+
+        anonymized_parameter_builder["anonymized_name"] = self.anonymize(
+            parameter_builder.get("name")
+        )
+
+        batch_request: Optional[dict] = parameter_builder.get("batch_request")
+        if batch_request:
+            anonymized_parameter_builder[
+                "anonymized_batch_request"
+            ] = self._batch_request_anonymizer.anonymize_batch_request(**batch_request)
+            logger.debug("Anonymized batch request in ParameterBuilder")
+
+        return anonymized_parameter_builder
+
+    def _anonymize_expectation_configuration_builders(
+        self, expectation_configuration_builders: List[dict]
+    ) -> List[dict]:
+        anonymized_expectation_configuration_builders: List[dict] = []
+
+        for expectation_configuration_builder in expectation_configuration_builders:
+            anonymized_expectation_configuration_builder: dict = (
+                self._anonymize_expectation_configuration_builder(
+                    expectation_configuration_builder
+                )
+            )
+            anonymized_expectation_configuration_builders.append(
+                anonymized_expectation_configuration_builder
+            )
+
+        return anonymized_expectation_configuration_builders
+
+    def _anonymize_expectation_configuration_builder(
+        self, expectation_configuration_builder: dict
+    ) -> dict:
+        anonymized_expectation_configuration_builder: dict = self.anonymize_object_info(
+            object_config=expectation_configuration_builder,
+            anonymized_info_dict={},
+            ge_classes=self._ge_expectation_configuration_builders,
+            runtime_environment={
+                "module_name": "great_expectations.rule_based_profiler.expectation_configuration_builder"
+            },
+        )
+
+        expectation_type: Optional[str] = expectation_configuration_builder.get(
+            "expectation_type"
+        )
+        if expectation_type in self._ge_expectation_types:
```

Comment:
Non blocking & up to you, but you might be able to refactor expectation anonymization into a public `ExpectationSuiteAnonymizer.anonymize_expectation()` method and then use here and in `ExpectationSuiteAnonymizer` to reduce duplication.

Reply:
might as well sneak it in here! let me respond to this once the change is up.

Reply:
done!